Telerik.JustMock/Mock.cs

```diff
@@ -62,6 +63,14 @@ public static INonPublicExpectation NonPublic
 			}
 		}
 
+		public static ILocalExpectation Local
+		{
+			get
+			{
+				return ProfilerInterceptor.GuardInternal(() => new LocalExpectation());
```

Comment:
@lyubomir-rusev, the similar comment like previous one is applicable here, I am not sure that guarding is even necessary. If you have some objections, please explain.